packages/components/src/disabled/stories/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Disabled from '../';
+import SelectControl from '../../select-control/';
+import TextControl from '../../text-control/';
+import TextareaControl from '../../textarea-control/';
+
+const meta: ComponentMeta< typeof Disabled > = {
+	title: 'Components/Disabled',
+	component: Disabled,
+	argTypes: {
+		as: { control: { type: null } },
+		children: { control: { type: null } },
+		ref: { control: { type: null } },
```

Comment:
I'm thinking it would be better to remove the ref from Control, any ideas?

![スクリーンショット 2022-08-18 11 45 56](https://user-images.githubusercontent.com/1908815/185281840-d6973e0b-6d6f-489e-af37-5ce41aa81b20.png)